13/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md

```diff
@@ -714,10 +714,14 @@ Are you looking to upgrade an Umbraco Cloud project from 9 to 10? Follow the gui
 
 <details>
 
-<summary>10.latest to version 11</summary>
+<summary>10.latest to the latest Umbraco version</summary>
 
 It might be necessary to delete all of the `bin` and `obj` directories in each of the projects of your solution. It has been observed that Visual Studio's "Clean Solution" option is sometimes not enough.
 
+You can upgrade from Umbraco 10 to the latest version directly. If you choose to skip upgrading to versions 11 and 12, you will no longer receive warning messages for obsolete features. However, if you do skip these versions, any breaking changes will no longer compile.
+
+It is recommended to upgrade to the closest [Long-term Support (LTS) major](https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/) version. For Umbraco 10 that is Umbraco 12. For Umbraco 12 that is Umbraco 13 and so on.
```

Comment:
@eshanrnh Just curious as to why it is recommended to upgrade to the closest LTS major and not just the latest LTS 🤔

Reply:
I think you have a point and it should state latest rather than closest as the dropdown heading.

Reply:
@jonat123 there is a reason for this. And that is if one wants to upgrade from v9 to v13 they first need to upgrade to the closest LTS which is v10 and then from there they can upgrade to v13. If v12 would have been LTS and they would have needed to upgrade from v9 to v10 and then v10 to v12 and then v13. This is what we have discussed internally.

Reply:
so when upgrading from v10, they can upgrade directly to v13 as it is lts however the next version they can upgrade directly from v13 is to next lts which is v17. So if one would want to migrate from v10 to v17 they first need to upgrade to v13 and then v17.